Mage.Verify failing on Unicode in multi-face cards

Current failure in Mage.Verify ([example](https://github.com/magefree/mage/actions/runs/30502094905/job/90743743528))

```
error, found unsupported unicode symbol in Glóin the Mighty // Easy Pickings - HOB
error, found unsupported unicode symbol in Glóin the Mighty // Easy Pickings - HOB
[INFO] 
[INFO] Standard error
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   VerifyCardDataTest.test_checkUnicodeCardNamesForImport:3588 Card name converters contains unsupported unicode symbols in 2 cards, see logs above
```

Digging into it, the failure is coming from;

```
        // check all converters
        Collection<String> errorsList = new ArrayList<>();
        MtgJsonService.sets().values().forEach(jsonSet -> {
            jsonSet.cards.forEach(jsonCard -> {
                if (jsonCard.isUseUnicodeName()) {
                    String inName = jsonCard.getNameAsUnicode();
                    String outName = CardNameUtil.normalizeCardName(inName);
                    String needOutName = jsonCard.getNameAsFace();
                    if (!outName.equals(needOutName)) {
                        // how-to fix: add new unicode symbol in CardNameUtil.normalizeCardName
                        errorsList.add(String.format("outName %s - needOutName %s", outName, needOutName));
                        errorsList.add(String.format("error, found unsupported unicode symbol in %s - %s", inName, jsonSet.code));
                    }
                }
            });
        });
```

The unicode character in question is already handled in https://github.com/magefree/mage/blob/50f7bee9a00391c2ae0cfacbcd0520d5dd7fe729/Mage/src/main/java/mage/cards/decks/CardNameUtil.java#L32

The problem is that right now, the code above is passing in `outName` as `Gloin the Mighty // Easy Pickings` and matching against `needOutName` as `Glóin the Mighty`

Currently AllPrintings.json data for these cards:

```
{
  "artist": "Colin Boyer",
  "artistIds": [
    "9c201dbe-db56-429a-87e6-189ea70c2632"
  ],
  "asciiName": "Gloin the Mighty // Easy Pickings",
  "availability": [],
  "borderColor": "black",
  "colorIdentity": [
    "R"
  ],
  "colors": [
    "R"
  ],
  "convertedManaCost": 4.0,
  "faceConvertedManaCost": 4.0,
  "faceManaValue": 4.0,
  "faceName": "Glóin the Mighty",
  "finishes": [
    "foil",
    "nonfoil"
  ],
  "foreignData": [],
  "frameVersion": "2015",
  "identifiers": {
    "mtgjsonV4Id": "aabaab11-50f2-5e8b-b92b-c0e0d44f3a87",
    "scryfallCardBackId": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "scryfallId": "5793b8eb-2fc5-454d-8fa2-20346fef167a",
    "scryfallIllustrationId": "2dc7b931-b20e-4561-9021-8a6c3cf021f6",
    "scryfallOracleId": "6505a53b-00db-44c4-8eaa-09a1a01451a4"
  },
  "language": "English",
  "layout": "adventure",
  "leadershipSkills": {
    "brawl": false,
    "commander": true,
    "oathbreaker": false,
    "pauper_commander": true,
    "predh": false
  },
  "legalities": {},
  "manaCost": "{3}{R}",
  "manaValue": 4.0,
  "name": "Glóin the Mighty // Easy Pickings",
  "number": "99",
  "otherFaceIds": [
    "812c1bf3-6030-5ba6-9acb-d8925f519c6f"
  ],
  "power": "4",
  "printings": [
    "HOB"
  ],
  "producedMana": [
    "R"
  ],
  "purchaseUrls": {},
  "rarity": "uncommon",
  "setCode": "HOB",
  "side": "a",
  "skuIds": {
    "foil": "531ecd31-130a-5e49-89e1-694bc52d442a",
    "nonfoil": "7a45d958-065a-5028-8fa0-87fa17af902e"
  },
  "subtypes": [
    "Dwarf",
    "Warrior"
  ],
  "supertypes": [
    "Legendary"
  ],
  "text": "At the beginning of your first main phase, add {R}{R}.",
  "toughness": "3",
  "type": "Legendary Creature — Dwarf Warrior",
  "types": [
    "Creature"
  ],
  "uuid": "b9025cf2-889c-5718-a92b-ccdec9ae21d5",
  "variations": [
    "1971debc-e875-5e13-980c-fc16c2d9b16b"
  ]
}
{
  "artist": "Colin Boyer",
  "artistIds": [
    "9c201dbe-db56-429a-87e6-189ea70c2632"
  ],
  "availability": [],
  "borderColor": "black",
  "colorIdentity": [
    "R"
  ],
  "colors": [
    "R"
  ],
  "convertedManaCost": 4.0,
  "faceConvertedManaCost": 3.0,
  "faceManaValue": 3.0,
  "faceName": "Easy Pickings",
  "finishes": [
    "foil",
    "nonfoil"
  ],
  "foreignData": [],
  "frameVersion": "2015",
  "identifiers": {
    "mtgjsonV4Id": "a282875a-fe03-5af4-a527-0442aa50ec3f",
    "scryfallCardBackId": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "scryfallId": "5793b8eb-2fc5-454d-8fa2-20346fef167a",
    "scryfallIllustrationId": "2dc7b931-b20e-4561-9021-8a6c3cf021f6",
    "scryfallOracleId": "6505a53b-00db-44c4-8eaa-09a1a01451a4"
  },
  "language": "English",
  "layout": "adventure",
  "legalities": {},
  "manaCost": "{2}{R}",
  "manaValue": 4.0,
  "name": "Glóin the Mighty // Easy Pickings",
  "number": "99",
  "otherFaceIds": [
    "b9025cf2-889c-5718-a92b-ccdec9ae21d5"
  ],
  "printings": [
    "HOB"
  ],
  "producedMana": [
    "R"
  ],
  "purchaseUrls": {},
  "rarity": "uncommon",
  "setCode": "HOB",
  "side": "b",
  "skuIds": {
    "foil": "68c41f80-4f1b-5d3d-ae0b-d02d84a8d369",
    "nonfoil": "29d0f0ad-5dba-587a-afa5-ca64cba4ea83"
  },
  "subtypes": [
    "Adventure"
  ],
  "supertypes": [],
  "text": "Easy Pickings deals 1 damage to each creature your opponents control. (Then exile this card. You may cast the creature later from exile.)",
  "type": "Sorcery — Adventure",
  "types": [
    "Sorcery"
  ],
  "uuid": "812c1bf3-6030-5ba6-9acb-d8925f519c6f",
  "variations": [
    "25490dee-5324-5a91-8164-92d1fce914ed"
  ]
}
{
  "artist": "Francisco Badilla",
  "artistIds": [
    "d876f06d-e6a6-4b0d-abe3-e7c9900ebf0c"
  ],
  "asciiName": "Gloin the Mighty // Easy Pickings",
  "availability": [],
  "borderColor": "black",
  "colorIdentity": [
    "R"
  ],
  "colors": [
    "R"
  ],
  "convertedManaCost": 4.0,
  "faceConvertedManaCost": 4.0,
  "faceManaValue": 4.0,
  "faceName": "Glóin the Mighty",
  "finishes": [
    "foil",
    "nonfoil"
  ],
  "foreignData": [],
  "frameVersion": "2015",
  "identifiers": {
    "mtgjsonV4Id": "53e78258-f295-586c-8fea-74c6551a138d",
    "scryfallCardBackId": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "scryfallId": "7c9bfc6e-648a-482b-9483-9e8538726f53",
    "scryfallIllustrationId": "952f2b9b-5ec4-4040-9d3a-db9d141483d7",
    "scryfallOracleId": "6505a53b-00db-44c4-8eaa-09a1a01451a4"
  },
  "isAlternative": true,
  "language": "English",
  "layout": "adventure",
  "leadershipSkills": {
    "brawl": false,
    "commander": true,
    "oathbreaker": false,
    "pauper_commander": true,
    "predh": false
  },
  "legalities": {},
  "manaCost": "{3}{R}",
  "manaValue": 4.0,
  "name": "Glóin the Mighty // Easy Pickings",
  "number": "227",
  "otherFaceIds": [
    "25490dee-5324-5a91-8164-92d1fce914ed"
  ],
  "power": "4",
  "printings": [
    "HOB"
  ],
  "producedMana": [
    "R"
  ],
  "purchaseUrls": {},
  "rarity": "uncommon",
  "setCode": "HOB",
  "side": "a",
  "skuIds": {
    "foil": "2806b740-91e6-5a28-853b-7e6f99237a5c",
    "nonfoil": "8c040f29-4862-53d8-b58d-1e873d535b7f"
  },
  "subtypes": [
    "Dwarf",
    "Warrior"
  ],
  "supertypes": [
    "Legendary"
  ],
  "text": "At the beginning of your first main phase, add {R}{R}.",
  "toughness": "3",
  "type": "Legendary Creature — Dwarf Warrior",
  "types": [
    "Creature"
  ],
  "uuid": "1971debc-e875-5e13-980c-fc16c2d9b16b",
  "variations": [
    "b9025cf2-889c-5718-a92b-ccdec9ae21d5"
  ]
}
{
  "artist": "Francisco Badilla",
  "artistIds": [
    "d876f06d-e6a6-4b0d-abe3-e7c9900ebf0c"
  ],
  "availability": [],
  "borderColor": "black",
  "colorIdentity": [
    "R"
  ],
  "colors": [
    "R"
  ],
  "convertedManaCost": 4.0,
  "faceConvertedManaCost": 3.0,
  "faceManaValue": 3.0,
  "faceName": "Easy Pickings",
  "finishes": [
    "foil",
    "nonfoil"
  ],
  "foreignData": [],
  "frameVersion": "2015",
  "identifiers": {
    "mtgjsonV4Id": "623cfa41-4ec9-5d8d-b8c9-e98e77c022cc",
    "scryfallCardBackId": "0aeebaf5-8c7d-4636-9e82-8c27447861f7",
    "scryfallId": "7c9bfc6e-648a-482b-9483-9e8538726f53",
    "scryfallIllustrationId": "952f2b9b-5ec4-4040-9d3a-db9d141483d7",
    "scryfallOracleId": "6505a53b-00db-44c4-8eaa-09a1a01451a4"
  },
  "isAlternative": true,
  "language": "English",
  "layout": "adventure",
  "legalities": {},
  "manaCost": "{2}{R}",
  "manaValue": 4.0,
  "name": "Glóin the Mighty // Easy Pickings",
  "number": "227",
  "otherFaceIds": [
    "1971debc-e875-5e13-980c-fc16c2d9b16b"
  ],
  "printings": [
    "HOB"
  ],
  "producedMana": [
    "R"
  ],
  "purchaseUrls": {},
  "rarity": "uncommon",
  "setCode": "HOB",
  "side": "b",
  "skuIds": {
    "foil": "b7d062fc-56ef-5da1-81f6-a04bec81e00d",
    "nonfoil": "cd440ce2-a511-5b2f-833a-337c6256ca27"
  },
  "subtypes": [
    "Adventure"
  ],
  "supertypes": [],
  "text": "Easy Pickings deals 1 damage to each creature your opponents control. (Then exile this card. You may cast the creature later from exile.)",
  "type": "Sorcery — Adventure",
  "types": [
    "Sorcery"
  ],
  "uuid": "25490dee-5324-5a91-8164-92d1fce914ed",
  "variations": [
    "812c1bf3-6030-5ba6-9acb-d8925f519c6f"
  ]
}
```

Need to poke a bit more at `MtgJsonCard` and what this test is trying to validate, but the issue seems to be coming down in part to:

```
    /**
     * @return single side name like Ice from Fire // Ice
     */
    public String getNameAsFace() {
        // return single side name
        return faceName != null ? faceName : (asciiName != null ? asciiName : name);
    }
```

returns just the unicode encoded face name

but also getNameAsUnicode just taking the full unicode card name, and not the unicode face name, normalising that and getting an ascii face name to compare against

--- a/Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java
+++ b/Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java
@@ -421,16 +421,16 @@ public class VerifyCardDataTest {
             for (MtgJsonCard refCard : refSet.cards) {
                 String cleanNumber = refCard.number.replaceAll("[\\D]", "");
                 if (cleanNumber.isEmpty()) {
-                    System.out.println("Found non-digit card number: " 
-                        + refSet.code + " - " 
-                        + refCard.getNameAsASCII() + " - " 
+                    System.out.println("Found non-digit card number: "
+                        + refSet.code + " - "
+                        + refCard.getNameAsASCII() + " - "
                         + refCard.number
                     );
                 }
                 if (cleanNumber.equals("0")) {
-                    System.out.println("Found zero card number: " 
-                        + refSet.code + " - " 
-                        + refCard.getNameAsASCII() + " - " 
+                    System.out.println("Found zero card number: "
+                        + refSet.code + " - "
+                        + refCard.getNameAsASCII() + " - "
                         + refCard.number
                     );
                 }
@@ -1972,7 +1972,7 @@ public class VerifyCardDataTest {
 
     // "copy" fails means that the copy constructor are not correct inside a card.
     // To fix those, try to find the class that did trigger the copy failure, and check
-    // that copy() exists, a copy constructor exists, and the copy constructor is right. 
+    // that copy() exists, a copy constructor exists, and the copy constructor is right.
     private void checkCardCanBeCopied(Card card1) {
         Card card2;
         try {
@@ -3575,7 +3575,7 @@ public class VerifyCardDataTest {
                 if (jsonCard.isUseUnicodeName()) {
                     String inName = jsonCard.getNameAsUnicode();
                     String outName = CardNameUtil.normalizeCardName(inName);
-                    String needOutName = jsonCard.getNameAsFace();
+                    String needOutName = jsonCard.getNameAsASCII();
                     if (!outName.equals(needOutName)) {
                         // how-to fix: add new unicode symbol in CardNameUtil.normalizeCardName
                         errorsList.add(String.format("error, found unsupported unicode symbol in %s - %s", inName, jsonSet.code));

